tests/api_tests/nodes/test_views.py

```diff
@@ -486,7 +526,7 @@ def test_update_public_project_logged_in(self):
         assert_equal(res.status_code, 403)
 
     def test_update_private_project_logged_out(self):
-        res = self.app.put_json(self.private_url, {
+        res = self.app.put(self.private_url, {
```

Comment:
Did you investigate why that happens? That looks like a possible regression because the test looks correct (with `put_json`).